velox/expression/ExprConstants.h

```diff
@@ -17,7 +17,7 @@
 
 namespace facebook::velox::expression {
 
-constexpr const char* kConjunt = "conjunct";
+constexpr const char* kConjunct = "conjunct";
```

Comment:
'conjunct' is not a function or special form name, is it? How is this used? Why do we need this constant?

Reply:
I see. Do we ned this? It is confusing to mix function names with just names.